apps/vscode/command_client/command_client.py

```diff
@@ -1,5 +1,6 @@
 import json
 import os
+import string
```

Comment:
I think this shouldn't be here. You seem to be using this as a type, but it isn't a type; the Python type for strings is `str`.

Reply:
Thank you for the heads up.